Userland/Applications/TodoList/TaskListModel.cpp

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2021, Arthur Brainville <ybalrid@ybalrid.info>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include "TaskListModel.h"
+
+#include <AK/JsonArray.h>
+#include <AK/JsonObject.h>
+#include <AK/JsonParser.h>
+#include <AK/JsonValue.h>
+
+#include <cstdio>
+
+void TaskListModel::load_from_disk(AK::String path)
```

Comment:
```suggestion
void TaskListModel::load_from_disk(StringView path)
```

Specifying the `AK::` namespace shouldn't be necessary in most cases because the classes are also exported into the global namespace.

Also, by using `StringView` here you can avoid copying the `String` object.